init.php

```diff
@@ -36,10 +36,11 @@
 require(dirname(__FILE__) . '/lib/Error/OAuth/UnsupportedResponseType.php');
 
 // API operations
-echo "bouh";
```

Comment:
Leftover from manual debugging 😨

Reply:
Oops. Totally missed that apparently, haha.